### DISCUSSION TITLE
Search button after annotating to make it easy to find and mark different cases.

### DISCUSSION THREAD

**Discussion by pederdm000** [tosdr/edit.tosdr.org#1067] [Ideas] [answered]
Search button after annotating on top of the case list to make it easy to find and mark different cases  when annotating.
![Capture45655](https://user-images.githubusercontent.com/20587711/113238879-272f6080-9299-11eb-838e-16c8f69a8a2e.PNG)

**Comment by JustinBack** [comment, marked as answer]
Added it to the roadmap https://git.tosdr.org/tosdr/roadmap/issues/4
